0-common/schema-ddl/project/plugins.sbt

```diff
@@ -1 +1 @@
-addSbtPlugin("me.lessis" % "bintray-sbt" % "0.3.0")
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.7")
```

Comment:
to create a fat jar. not sure if really needed but just in case.

Reply:
No, we don't need assembly. Basically, we have two types of assets:

* Applications - assets that are going to be invoked directly by user and distributed as executable binaries. We use `sbt-assembly` to package and [release-manager](https://github.com/snowplow/release-manager/) (app that works with `release.yml`) to publish them
* Libraries - assets that are going to be included as dependency in applications and that have no value as stand-alone binaries. We use `sbt-bintray` to publish them on Maven.

`schema-ddl` is a typical library, so we don't use sbt-assembly and we do need `bintray-sbt` for it.